### ISSUE TITLE
OSO / Kariba Brand comparison

### ISSUE THREAD

**Issue by ciciaran** [opensource-observer/oso#2594]

**Comment by ryscheng** [comment]
@ciciaran do you mind linking to the work product here so I can take a look? In general would be helpful for closing out any issues. Thanks!

**Comment by ciciaran** [comment]
Yes here's a link to [the pdf ](https://drive.google.com/file/d/1ruxG86gZOW8WanZG_gtOswt9WH_ggKtz/view?usp=sharing)
